cronJobs.js

```diff
@@ -0,0 +1,51 @@
+var CronJob = require('cron').CronJob;
+var moment = require('moment');
+
+module.exports = async function(db) {
+  new CronJob('0 9 * * *', async function() { // replace with * * * * * to run every minute for testing
+    const now = new Date();
+    const weekFromNow = new Date();
+    weekFromNow.setDate(now.getDate() + 7);
+    const requirementsAboutToExpire = await db.get('projectRequirementsCollection').find({
+      $and: [
+        {
+          solvedate: {
+            $gt: now,
+            $lt: weekFromNow
+          }
+        },
+        {
+          $or: [
+            {
+              status: {
+                $eq: "OPEN"
+              }
+            },
+            {
+              status: {
+                $eq: "PENDING"
+              }
+            }
+          ]
+        }
+      ]
+    });
+  
+    if (requirementsAboutToExpire && requirementsAboutToExpire.length) {
+      const messages = requirementsAboutToExpire.map((a) => {
```

Comment:
Can you make a method to return the messages formatted?

Reply:
y